### PR TITLE
chore(comments): Click to edit

### PR DESCRIPTION
**Pull request by ZaymonFC**
- Resolves https://github.com/dxos/dxos/issues/5462

This also fixes an issue where comment edits were not being persisted.

### CODE REVIEW

**Review by thure — APPROVED** [review]
lgtm!

File: packages/apps/composer-app/src/playwright/comments.spec.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ test.describe('Comments tests', () => {
     await expect(Thread.getThreads(host.page)).toHaveCount(1);
   });
 
-  test('edit message', async () => {
+  test.only('edit message', async () => {
```
Comment (wittjosiah) [inline packages/apps/composer-app/src/playwright/comments.spec.ts:44]: .only 👀

File: packages/apps/composer-app/src/playwright/comments.spec.ts [hunk 2]
```diff
@@ -57,12 +57,24 @@ test.describe('Comments tests', () => {
...
+    await expect(messageTextbox).toContainText(messageText);
+
+    const editButton = host.page.getByTestId('thread.message.edit');
+    await editButton.click();
 
     const editedText = 'Edited';
-    await message.fill('');
-    await message.fill(editedText);
+
+    for (let i = 0; i <= messageText.length; i++) {
```
Comment (wittjosiah) [inline packages/apps/composer-app/src/playwright/comments.spec.ts:70]: did `fill('')` stop working?

Comment (ZaymonFC) [reply packages/apps/composer-app/src/playwright/comments.spec.ts:70]: It auto-focuses after clicking the edit button which seemed to break it.

File: packages/apps/composer-app/src/playwright/comments.spec.ts (hunk 1 above)
Comment (ZaymonFC) [reply packages/apps/composer-app/src/playwright/comments.spec.ts:44]: Good catch.

**Pull request merged**